tools/xcodeproj-shims/json-util.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/python
+import sys, json; 
+j = json.load(sys.stdin)
+queue = [j]
+
+# Print all leaf nodes of a given json via stdin
```

Comment:
should we have a more desciptive file name, since this isn't a generic json utility and rather only does one specific thing?

Reply:
Agree, replaced all references of file names to `print-json-leaf-nodes`